app/com/m3/octoparts/repository/ConfigsRepository.scala

```diff
@@ -25,14 +25,34 @@ trait ConfigsRepository {
   def findParamById(id: Long): Future[Option[PartParam]]
 
   /**
-   * Find a single ThreadPoolConfig by id
+   * Find a single ThreadPoolConfig by id.
+   *
+   * All retrieved items are populated with their [[HystrixConfig]]
    */
-  def findThreadPoolConfigById(id: Long): Future[Option[ThreadPoolConfig]]
+  def findThreadPoolConfigById(id: Long): Future[Option[ThreadPoolConfig]] = findAllConfigs().map {
```

Comment:
Point taken. I tried with joins too but failed to get the ORM to behave the way I wanted. 

So I thought, we are already doing a full retrieve on other admin pages, so why not here as well?